com.unity.render-pipelines.universal/Runtime/ForwardLights.cs

```diff
@@ -353,10 +359,11 @@ public void Setup(ScriptableRenderContext context, ref RenderingData renderingDa
 
                 SetupShaderLightConstants(cmd, ref renderingData);
 
+                bool lightCountCheck = (m_AdditionalLightsAlwaysEnabled && renderingData.lightData.maxPerObjectAdditionalLightsCount > 0) || additionalLightsCount > 0;
```

Comment:
can `additionalLightsCount` be changed at runtime and you get into a situation that the keyword and stripped and at runtime you try to enable it?

Reply:
add additional check to prevent this